entrypoint.sh

```diff
@@ -4,6 +4,6 @@ set -eo pipefail
 
 npx depcheck@"${INPUT_VERSION}" --ignores="${INPUT_IGNORES}" "${INPUT_DIR}" 2>&1 && exit_status=$? || exit_status=$?
 
-echo "::set-output name=exit_code::$exit_status"
+echo "exit_code=$exit_status" >> $GITHUB_OUTPUT
```

Comment:
We can use double quotes to [prevent globbing and word splitting](https://www.shellcheck.net/wiki/SC2086)

```suggestion
echo "exit_code=$exit_status" >> "$GITHUB_OUTPUT"
```

Reply:
Fixed